src/web/components/StickyBottomBanner/StickyBottomBanner.tsx

```diff
@@ -18,63 +22,96 @@ const getBannerLastClosedAt = (key: string): string | undefined => {
     return (item && JSON.parse(item).value) || undefined;
 };
 
+const buildNewCmpBannerConfig = () => {
```

Comment:
might be helpful to specify the type here
`(): Banner => ...`

Reply:
Great idea. Resolved in 6a6c7fd0fac3daec8b0184899dcb843585a28165.